src/install.rst

```diff
@@ -184,6 +184,23 @@ Set up a Nextstrain runtime
             .. include:: snippets/nextstrain-setup-conda.rst
 
 
+   .. group-tab:: Singularity
+
+      Singularity is a container system freely-available for Linux platforms. It is commonly available on institutional HPC systems as an alternative to Docker, which is often not supported on such systems.
```

Comment:
As @j23414 [suggested elsewhere](https://github.com/nextstrain/cli/pull/353#issuecomment-1919803164), it's probably worth noting here that Singularity refers to both forks: SingularityCE and Apptainer (at least as long as a `singularity` executable is available). There's language to crib from at https://docs.nextstrain.org/projects/cli/en/stable/runtimes/singularity/?highlight=Apptainer.

Reply:
[Updated](https://github.com/nextstrain/docs.nextstrain.org/compare/cb4e30fa1d26fbd8761622785db5dc20a91fdc64..0961b9eb3564f7f776c1791c8a27e3e5e5f29567) to mention SingularityCE/Apptainer.